_examples/http-server/west_bindings_test.go

```diff
@@ -0,0 +1,2015 @@
+// Code generated by west-bindgen-go DO NOT EDIT
```

Comment:
Out of curiosity, what purpose does the bindings test serve in the consumer code?

Reply:
This file contains Wasm import replacements, which allows applications to compile for native targets for testing. It is what enables *west*

By default it's generated in `<pkg>_test` package (unless it's `main`) and the file is suffixed with `_test.go`. This all is done to ensure this logic never appears in actual release implementation